content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md

```diff
@@ -1,22 +1,21 @@
 ---
 title: Coarse Parallel Processing Using a Work Queue
-min-kubernetes-server-version: v1.8
 content_type: task
 weight: 20
 ---
 
 
 <!-- overview -->
 
-In this example, we will run a Kubernetes Job with multiple parallel
+In this example, you will run a Kubernetes Job with multiple parallel
 worker processes.
 
 In this example, as each pod is created, it picks up one unit of work
 from a task queue, completes it, deletes it from the queue, and exits.
 
 Here is an overview of the steps in this example:
 
-1. **Start a message queue service.**  In this example, we use RabbitMQ, but you could use another
+1. **Start a message queue service.**  In this example, you use RabbitMQ, but you could use another
```

Comment:
nit: Could you instead say, The example uses RabbitMQ, but you could choose <another messaging broker>.

Reply:
Yep, that also works.